easylinks/src/main/webapp/manage.html

```diff
@@ -8,6 +8,8 @@
     <script src="script.js"></script>
   </head>
   <body onload="getLinks()">
+    <button onclick="location.href='/index.html'" type="button" class="redirectButton">
```

Comment:
maybe make this a function in js? onlick="redirectToHomePage()"

Reply:
Done.